skymap_scanner/recos/__init__.py

```diff
@@ -49,13 +67,12 @@ def get_all_reco_algos() -> List[str]:
     ]
 
 
-def get_reco_interface_object(name: str) -> RecoInterface:
+def get_reco_interface_object(name: str) -> type[RecoInterface]:
```

Comment:
I think this syntax is py3.11 only. You'll need to import `Type`

Reply:
According to docs, `builtins.type` is supported since 3.9 and `Type` is actually deprecated.

Reply:
nice!